sdk/python/tests/integration/conftest.py

```diff
@@ -9,14 +12,30 @@
 from tests.utils.auth_permissions_util import setup_permissions_on_keycloak
 
 logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
any reason to set log level as `INFO` as default is `warning` https://github.com/feast-dev/feast/blob/master/sdk/python/feast/cli.py#L84

Reply:
This log is only for integration tests to display info logs. Keeping it info helped to display the important logs during the test runs. These logs purged after the new run per PR only. It does not harm to keep it Info but also if you want we can change it WARN as well.

Reply:
we can keep Feast default log level, until unless it's required to see the info message.

Reply:
I can update in the next PR as this is merged.